src/Uno.Toolkit.Skia.WinUI/Controls/Shadows/ShadowContainer.Paint.cs

```diff
@@ -78,7 +78,7 @@ public static SKShadow From(Shadow shadow, float width, float height, float corn
 		}
 	}
 
-	private ShadowInfos[] _shadowInfoArray = new[] { ShadowInfos.Empty };
+	private ShadowInfos[] _shadowInfoArray = Array.Empty<ShadowInfos>();
```

Comment:
This changes the meaning of the code. Is it intentional?

Reply:
Yes, it's. If you look into the code, this a `private` field and is used just to hold the `oldShadowCollection`, basically to see if something changed and should invalidate something. So, AFAIK, it's a safe change